src/main/kotlin/org/wycliffeassociates/otter/jvm/persistence/repositories/WorkbookRepository.kt

```diff
@@ -40,54 +41,53 @@ class WorkbookRepository(
             title = bookCollection.titleKey,
             sort = bookCollection.sort,
             chapters = constructBookChapters(bookCollection),
-            subtreeResources = resourceRepository.getSubtreeResourceInfo(bookCollection)
+            subtreeResources = db.getSubtreeResourceInfo(bookCollection)
         )
     }
 
     private fun constructBookChapters(bookCollection: Collection): Observable<Chapter> {
-        val collections = collectionRepository
-            .getChildren(bookCollection)
-            .flattenAsObservable { list -> list.sortedBy { it.sort } }
-
-        val chapters = collections
-            .concatMapEager { constructChapter(it).toObservable() }
-
-        return chapters.cache()
+        val coldObservable = Observable.fromCallable {
```

Comment:
how about "sortedChapters" as a name here?

Reply:
most things we're doing with rx are cold observables, so my thought was that the current name doesn't provide a lot of information regarding what the observable is doing

Reply:
This little dance where I'm wrapping observable inside observable is in order to keep them from going hot. Some operations like flatmaps seem to make them go hot (revealed by the tests in this pr). So the purpose of this wrapper is to keep everything *cold* and be lazy. I can see it's a weird name though and can rename it.

Reply:
I couldn't fix this by switching to map or concatMap from concatMapEager. However, the Observable.defer function does what I needed more cleanly, so I reverted this change and implemented that.